networking_monitoring.prolific

```diff
@@ -97,6 +97,99 @@ L: security
+
+### How?
+**Setup**
+1. You should have two apps pushed named Boots and Swiper.
+1. There should be no c2c network policies (`cf network-policies` and `cf remove-network-policies --help`).
+
+**Curl swiper from boots**
+1. Get the overlay IPs of boots and swiper.
+ ```
+cf ssh boots -c "env | grep CF_INSTANCE_INTERNAL_IP"
+cf ssh swiper -c "env | grep CF_INSTANCE_INTERNAL_IP"
+ ```
+1. Ssh onto boots.
+ ```
+cf ssh boots
+ ```
+1. Continually try to curl swiper from boots
+ ```
+watch -n 15 curl SWIPER_OVERLAY_IP:8080
+ ```
+
+**Look at those packets**
+1. In another terminal, ssh onto the Diego Cell where boots is running and become root (see the help section if you don't know how to do this).
+1. Run `tcpdump`.
+    Ahhhhh too much information! ctrl+c! ctrl+c!  On a Diego Cell there are many packets being sent around, and tcpdump gives information about ALL OF THEM. You need to figure out a way to filter this overwhelming stream of information.
+1.  Filter by packets where the source IP is BOOTS_OVERLAY_IP and where the destination IP is SWIPER_OVERLAY_IP over any interface. (Interested in learning more about network interfaces? Contact Amelia Downs about CF Networking Onboarding week! Or google it.)
+ ```
+tcpdump -n src BOOTS_OVERLAY_IP and dst SWIPER_OVERLAY_IP -i any
+ ```
+ Hey! Those are your packets!
+
+ Record the packets you see here from one curl.
+ ```
+# PUT TCPDUMP OUTPUT HERE
+ ```
+
+If swiper was successfully responding, then you should also see packets being sent as a response in the opposite direction.
+
+1. See that no packets are being sent back from Swiper to Boots
+  ```
+tcpdump -n src SWIPER_OVERLAY_IP and dst BOOTS_OVERLAY_IP -i any
+ ```
+
+**Add c2c policy**
+1. Add c2c policy to allow traffic from Boots to Swiper (`cf add-network-policy --help`)
+1. Continually try to curl Swiper from Boots
+
+**Look at those packets**
+1. Look for packets from Boots to Swiper
+ ```
+tcpdump -n src BOOTS_OVERLAY_IP and dst SWIPER_OVERLAY_IP -i any
+ ```
+ Record the packets you see here from one curl.
+
+ ```
+ PUT TCPDUMP OUTPUT HERE
+ ```
+How are these packets different from before?
+
+1. Look for packets from Swiper to Boots
+ ```
+tcpdump -n src SWIPER_OVERLAY_IP and dst BOOTS_OVERLAY_IP -i any
+ ```
+
+### Expected Result
+You should see packets being sent in response from Swiper to Boots. You should see `200 OK`.
+
+### Help
+How to determine what Diego Cell your app is running on:
+```
+# First determine the IP of the Diego Cell
+cf ssh APP_NAME  -c "env | grep CF_INSTANCE_IP"
+
+# Look at bosh output to see which Diego Cell has that IP
+bosh instances
+```
+
+### Resources
+[tcpdump man page](https://www.tcpdump.org/manpages/tcpdump.1.html)
+[helpful common tcpdump commands](https://www.rationallyparanoid.com/articles/tcpdump.html)
+[debugging non-c2c traffic in CF](https://github.com/cloudfoundry/cf-networking-release/blob/develop/docs/troubleshooting.md#debugging-non-c2c-packets)
+
+L: security
```

Comment:
This must mean something but I am not sure what 🤔

Reply:
If you are referencing the "debugging non-c2c traffic"  - that was left over from the tcpdump story around ASGs. I have removed this link. 

If you are referencing the label - all of the other stories in this series have that label so I kept it to be consistent. I'm happy to remove it, if you think that's best.

Reply:
Oh sorry for the ambiguity. I was referring to the `L: security`.

TIL! I didn't know that was part of the DSL in here. Thanks and feel free to keep it! (I definitely don't know everything, just doing what I can to help with PRs in this repo :P)